public/app/plugins/datasource/jaeger/datasource.test.ts

```diff
@@ -39,6 +39,8 @@ jest.mock('@grafana/runtime', () => ({
 }));
 
 describe('JaegerDatasource', () => {
+  const defaultSearchRangeParams = `start=${Number(defaultQuery.range.from) * 1000}&end=${Number(defaultQuery.range.to) * 1000}`;
```

Comment:
The tests are failing, you'll need to move `defaultQuery` up in the file. I'd suggest putting it just bellow `export const backendSr...`